<launch>
	<!-- Load additional parameters from cfg/params.yaml -->
	<rosparam command="load" file="$(find lab6)/cfg/params.yaml" ns="/motion_planning/" />
	
	<node pkg="lab6" type="path_planner.py" name="Trajectory_search" output="screen">
	</node>
</launch>
